### PR TITLE
Node selector for resources

### PR DESCRIPTION
**Pull request by UnderGreen**
Scale sentinels to 3 for 3x and more keeper nodes.

### CODE REVIEW

File: images/bootstrap/bootstrap.sh [hunk 1]
```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 
 /usr/local/bin/stolonboot -sentinels 1 -rpc 1
+
+if [[ $(kubectl get nodes -l stolon-keeper=yes -o name | wc -l) -eq 3 ]]
```
Comment (sofuture) [inline images/bootstrap/bootstrap.sh]: this should be `>= 3`

Comment (sofuture) [inline images/bootstrap/bootstrap.sh]: Do you need `--show-headers=false` or does the `-o name` handle that?

Comment (UnderGreen) [inline images/bootstrap/bootstrap.sh]: 1. Sure. Just have copied old condition from console. Updated.
2. Yes, it shows only node name line by line.

Comment (UnderGreen) [inline images/bootstrap/bootstrap.sh]:
```bash
$ kubectl get nodes -l stolon-keeper=yes -o name
node/172.28.128.13
node/172.28.128.14
node/172.28.128.15
```

**Comment by UnderGreen** [comment]
@sofuture I've did changes, which you asked.

**Pull request merged**